krkn/scenario_plugins/node_actions/aws_node_scenarios.py

```diff
@@ -366,8 +365,8 @@ def node_reboot_scenario(self, instance_kill_count, node, timeout):
                     "Rebooting the node %s with instance ID: %s " % (node, instance_id)
                 )
                 self.aws.reboot_instances(instance_id)
-                nodeaction.wait_for_unknown_status(node, timeout, self.kubecli, affected_node)
-                nodeaction.wait_for_ready_status(node, timeout, self.kubecli, affected_node)
+                self.kubecli.wait_for_unknown_status(node, timeout, affected_node)
```

Comment:
these functions are not in the krkn-lib repo. They need to stay using nodeaction 
should stay as: 
`nodeaction.wait_for_unknown_status(node, timeout, self.kubecli, affected_node)`